src/graphql.ts

```diff
@@ -0,0 +1,204 @@
+import {Document, FragmentDefinition} from 'graphql';
```

Comment:
`DocumentNode` and `FragmentDefinitionNode` if you're using GraphQL ~0.8.0: https://github.com/apollostack/apollo-client/pull/934